.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "Pillow"]
 	path = Pillow
-	url = git://github.com/python-Pillow/Pillow.git
+	url = git://github.com/fdintino/Pillow.git
```

Comment:
I presume this is just temporary to demonstrate that the PR passes?

Reply:
Yes, exactly. That's also why I made it a separate commit, so that it's easy to revert / rebase away.